pkg/controller/restore-operator/sync.go

```diff
@@ -136,9 +136,12 @@ func (r *Restore) prepareSeed(er *api.EtcdRestore) (err error) {
 	}()
 
 	cs := er.Spec.ClusterSpec
+	if err := cs.Validate(); err != nil {
+		return fmt.Errorf("invalid cluster spec. please fix the following problem with the cluster spec: %v", err)
```

Comment:
this is not a user facing error message. do not need to be this verbose. just invalid cluster spec: %v, error